content/changelog/2024-06-14.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Schema Diff in the CLI, a redesigned branch page, support for pgvector 0.7.1, and more
+---
+
+### Improved automation with Schema Diff in the CLI
+
+You can now integrate the Neon CLI’s `schema-diff` feature into your CI/CD pipelines, letting you compare schemas between branches at any point in their history &#8212; useful for making sure that only the intended schema changes are promoted, maintaining consistent and controlled deployments.
+
+Get started by using the command in your terminal:
+
+```bash
+neonctl branches schema-diff [base-branch] [compare-source[@(timestamp|lsn)]]
+```
+
+For example, this command compares the current schema state between the `main` branch and the development branch `dev/alex`:
+
+```bash
+neonctl branches schema-diff main dev/alex
+```
+
+Sample output of the `schema-diff` command might look like this:
+
+```sql
```

Comment:
```suggestion
```diff
```

Reply:
I think the language code here needs to be "diff" for the inline `[!code ++] [!code --] `to take effect.